sdk/tests/client/signing/nft.rs

```diff
@@ -160,7 +160,7 @@ async fn nft_reference_unlocks() -> Result<()> {
 
     let conflict = verify_semantic(&prepared_transaction_data.inputs_data, &tx_payload, current_time)?;
 
-    if conflict != ConflictReason::None {
+    if conflict != TransactionFailureReason::None {
```

Comment:
Just thinking that a `TransactionFailureReason::is_none` method could be useful

Reply:
Or `verify_semantic()` just returns `Option<TransactionFailureReason>` and we remove `TransactionFailureReason::None`

Reply:
If there is no other place where `TransactionFailureReason::None` would be used, sure we can do that

Reply:
Removed